apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/HHAClaimTransformerV2Test.java

```diff
@@ -197,15 +202,18 @@ public void shouldNotSetBillablePeriodWithNullClaimQueryCode() throws Exception
     String json = parser.encodeResourceToString(genEob);
     eob = parser.parseResource(ExplanationOfBenefit.class, json);
 
-    // We just want to make sure it is not set
-    assertNull(eob.getBillablePeriod());
```

Comment:
There should still be a billable period, we are only concerned that the billable period does not have the claim query code.